src/test/java/neculaclaudiu/tests/seleniumpart2/HerokuForms.java

```diff
@@ -0,0 +1,96 @@
+package neculaclaudiu.tests.seleniumpart2;
+
+import neculaclaudiu.driver.ChromeDriverManager;
+import neculaclaudiu.utils.FileUtils;
+import org.openqa.selenium.*;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.interactions.Actions;
+
+import java.util.List;
+
+public class HerokuForms {
+    static ChromeDriver driver = null;
+    public static void main(String[] args){
+        navigateToHerokuHomePage();
+        try{
+            openHtmlFormExamplePage();
+            fillInputZones();
+            scrollToSubmitButton();
+            fillInputZones();
+            clickToChooseFileButton();
+            clickToAllCheckBoxItems();
+            clickToAllRadioButtons();
+            clickToSubmitButton();
+            nonExistentElement();
+        } catch (NoSuchElementException e){
+            FileUtils.takeScreenshot(driver, "htmlForms");
+        }finally {
+            closeBrowser();
+        }
+
+
+    }
+    public static void navigateToHerokuHomePage() {
+        driver = ChromeDriverManager.createChromeDriver();
+        driver.get("https://testpages.herokuapp.com/styled/index.html");
+        System.out.println("Am deschis index page-ul");
+    }
+    public static void openHtmlFormExamplePage() {
+        WebElement htmlFormLink = driver.findElement(By.id("htmlformtest"));
+        htmlFormLink.click();
+        System.out.println("Am deschis pagina cu HTML");
+    }
+
+    public static void closeBrowser(){
+        driver.quit();
+        System.out.println("am inchis browser-ul");
+    }
+    public static void scrollToSubmitButton(){
+        WebElement submitButton = driver.findElement(By.cssSelector("input[type=\"submit\"]"));
+        Actions actions = new Actions(driver);
+        actions.moveToElement(submitButton).build().perform();
+        System.out.println("Am dat scroll la submitButton");
+    }
+    public static void fillInputZones(){
+        WebElement username = driver.findElement(By.cssSelector("input[name=\"username\"]"));
+        username.sendKeys("claudiu357");
+
+        WebElement password = driver.findElement(By.cssSelector("input[name=\"password\"]"));
+        password.sendKeys("11122233345667");
+
+        WebElement comments = driver.findElement(By.cssSelector("textarea[name=\"comments\"]"));
+        comments.sendKeys("Lalalalalalaalalal");
+    }
+    public static void clickToChooseFileButton(){
+        try{
+            WebElement chooseFileButton = driver.findElement(By.cssSelector("input[name=\"filename\"]"));
+            chooseFileButton.sendKeys("C://path//to//file.jpg");
```

Comment:
If you want to see a way of sending a file from a relative location, you can look into https://github.com/SkillBrain-com/Procyon-QAC-G1-13.10.22/pull/40, https://github.com/SkillBrain-com/Procyon-QAC-G1-13.10.22/pull/35, https://github.com/SkillBrain-com/Procyon-QAC-G1-13.10.22/pull/34 or https://github.com/SkillBrain-com/Procyon-QAC-G1-13.10.22/pull/33.